<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>求是图书管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>401</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#00aaff;&quot;&gt;求是图书管理系统&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic; color:#02c8ff;&quot;&gt;Q&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-style:italic; color:#02c8ff;&quot;&gt;iushi &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic; color:#02c8ff;&quot;&gt;L&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-style:italic; color:#02c8ff;&quot;&gt;ibrary &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic; color:#02c8ff;&quot;&gt;M&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-style:italic; color:#02c8ff;&quot;&gt;anagement &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic; color:#02c8ff;&quot;&gt;S&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-style:italic; color:#02c8ff;&quot;&gt;ystem&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_log_Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_user_status_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="main_userinfo_Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>我的信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_bookSearch_Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>图书查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_returnBook_Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>管理还书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_userManagement_Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>用户管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_manageBook_Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>管理图书</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="action_log">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
